src/worker/extensions/bouncer/index.js

```diff
@@ -325,6 +327,10 @@ async function handleBouncerCommand(event) {
             netUpdates.tls = (tags.tls === '1');
         }
 
+        if (tags.tlsverify) {
```

Comment:
Should be checking if it's not undefined? If it's been set to false then it would never get updated in the db

Reply:
`"0"` is truthy

Reply:
Ah my bad, I was mixing it up with the var further up